src/ui/pages/DashboardPage.jsx

```diff
@@ -21,7 +22,7 @@ function Dashboard() {
             <Grid item xs={4}>
               <GraphComponent>
                 <h1>Graph 1</h1>
-                <Text> Hola</Text>
+                <CircleGraph number={90} text="Puntuación" />
```

Comment:
We are going to get an endpoint to get all of the numbers, if you want wirte placeholder numbers write them inside the component